### ISSUE TITLE
HumanControlBoard fails to attach to iHumanState

### ISSUE THREAD

**Issue by lrapetti** [robotology/human-dynamics-estimation#364]
Trying to attach to a device exposing `iHumanState` interface is failing with the following:

```sh
...
[INFO] HumanControlBoard : Attaching device human_state_provider
[ERROR] HumanControlBoard : Unable to attach human_state_provider as one of the supported interfaces
[INFO] HumanControlBoard : Attaching device human_dynamics_estimator
[ERROR] yarprobotinterface intercepted a segmentation fault caused by a faulty plugin:
...
```

**Comment by lrapetti** [comment]
The problem seems to be a leftover from the fix implemented in https://github.com/robotology/human-dynamics-estimation/pull/347#issuecomment-1700843085, and it is fixed by https://github.com/robotology/human-dynamics-estimation/pull/365

**Comment by lrapetti** [comment]
> The problem seems to be a leftover from the fix implemented in [#347 (comment)](https://github.com/robotology/human-dynamics-estimation/pull/347#issuecomment-1700843085), and it is fixed by #365

PR merge, the issue has been fixed.
